### DISCUSSION TITLE
Should pacstall be the AUR for Ubuntu and Debian?

### DISCUSSION THREAD

**Discussion by Zahrun** [pacstall/pacstall#986] [Ideas]
Hi there,

"Pacstall - The AUR for Ubuntu" is how the package manager is branded in several places.
However, pacstall also supports Debian for the most part.
Should this be communicated more clearly and largely?

**Comment by Elsie19** [comment]
"Pacstall ~ The AUR for Ubuntu" is our tagline. It's meant to be short, recognizable, and something that people will search up and get an exact answer to. We have a section on the FAQ that explains that a lot of packages will work on Debian also, but that's just a happy coincidence, and Ubuntu is our primary target.

**Reply by Zahrun** [reply]
Ok, I understand the use of a nice tagline and a clear focus for the project.
I was just thinking that since we test all packages on Debian for each pr, and if something is not compatible with debian, we put that information in the script, maybe the word debian could appear in the README of https://github.com/pacstall/pacstall and on [pacstall.dev](https://pacstall.dev/).
I think it is really nice that pacstall pretty much work on all debian and ubuntu-based distros. Some Debian users might think that they cannot use pacstall because it’s meant for Ubuntu.
I am fine with the section in the FAQ, I just think it is not so apparent at first sight. I thought we could bring up more, as a feature, an asset of the project, and draw more people in.
But I am also totally fine if you keep it the way it is 👍
